homeassistant/components/sensor/uptimerobot.py

```diff
@@ -0,0 +1,91 @@
+"""A component to monitor Uptime Robot monitors.
+
+For more details about this component, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.uptimerobot
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_API_KEY
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['pyuptimerobot==0.0.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_TARGET = 'target'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_KEY): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Uptime Robot sensors."""
+    from pyuptimerobot import UptimeRobot
+
+    up_robot = UptimeRobot()
+    apikey = config.get(CONF_API_KEY)
+    monitors = up_robot.getMonitors(apikey)
+
+    devices = []
+    if not monitors or monitors.get('stat') != 'ok':
+        error = monitors.get('error', {})
+        _LOGGER.error(error.get('message', 'Something terrible happend :('))
```

Comment:
Please keep the error log message informative and kind of professional. The latter message here is too broad and breaks the log style of home assistant. Smilies are also out of style.

Sorry for sounding boring.

Reply:
I'll convert it to an binary_sensor and give a proper log entry :)